DTIPrep/DTIPrepRegister.pl

```diff
@@ -1368,22 +1542,28 @@ sub register_nrrd {
 }   
 
 
+=pod
 
+=head3 register_Preproc($mri_files, $dti_file, $data_dir, ...)
 
-
-=pod
 Gather all DTIPrep preprocessed files to be registered in the database
-and call register_DTIPrep_files on all of them. Will register first the 
-nrrd file and then the minc file for each scan type.
-Inputs:  - $mri_files       = hash containing all DTI output information
-         - $dti_file        = native DTI file that was processed (that will be used as a key for $mri_files)
-         - $data_dir        = data_dir defined in the config file
-         - $pipelineName    = name of the pipeline used to preprocess data (DTIPrepPipeline)
-         - $toolName        = name and version of the tool used to preprocess data
-         - $process_step    = processing step ('Preproc' or 'Postproc') depending on the processed file
-         - $proc_file       = key to the processed file to be registered ('QCed', 'QCed2'...)
-Outputs: - $registered_minc = path to the minc file that was registered
+and call C<&register_DTIPrep_files> on all of them. Will register first the
```

Comment:
and _calls_....

Gather above should also be replaced by _Gathers_

Reply:
done